internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer.go

```diff
@@ -301,7 +301,7 @@ func (*Sequencer) Reset(r runtime.Runtime, in runtime.ResetOptions) []runtime.Ph
 			"leave",
 			LeaveEtcd,
 		).AppendList(
-			stopAllPhaselist(r, withKexec),
+			withErrorHandler(stopAllPhaselist(r, withKexec), forceCleanup),
```

Comment:
this feels wrong to me, as it adds `forceCleanup` to each task hiding the original error?

I meant adding `forceCleanup` as a step here before actually resetting things